### ISSUE TITLE
x11-dl compilation eats all memory on nightly + Linux

### ISSUE THREAD

**Issue by Gekkio** [emoon/rust_minifb#39]
For some reason, on Linux and Rust nightly x11-dl 2.2 never compiles successfully and eats all the memory (got 16 GB on this machine). Upgrading to x11-dl 2.14 seems to fix the issue, so maybe that dependency could be upgraded?

```shell
$ uname -srvmpio
Linux 4.10.0-28-generic #32-Ubuntu SMP Fri Jun 30 05:32:18 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by emoon** [comment]
Hi, Thanks for the report. I guess it's some issue inside `x11-dll`. I will fix it.

**Comment by emoon** [comment]
0.9.2 has now been published that uses 2.14
